### ISSUE TITLE
关于训练用的训练集和测试集来源问题

### ISSUE THREAD

**Issue by ARnnn** [kyoukuntaro/FCSN_for_ChangeDetection_IGARSS2018#1]
我按照你的make_dataset.py处理完以后，得到的是一些.npy文件，并没有划分为训练集和测试集，在train.py文件中的测试集是怎么来的，是将原来data文件夹中的.npy文件自己划为训练集和测试集吗

**Comment by kyoukuntaro** [comment]
Thank you for your question.
Indeed, make_dataset.py make only one dataset including both of train and test dataset.
I divided it manually.

**Comment by ARnnn** [comment]
Thank you for your response.
In addition,how do you manually generate the test suite? Do you merge the two corresponding pictures in the files contained in the test suite into six channels?

**Comment by kyoukuntaro** [comment]
>Do you merge the two corresponding pictures in the files contained in the test suite into six channels?

Yes, I think it is a good choice. Is that the answer you were hoping for?

**Comment by ARnnn** [comment]
Why didn't this code end up with a  .h5 model?

**Comment by goosey-geo** [comment]
Hello, I do not know what to do when the training progress finished, can you give me some advice? I'm a really new one for deep learning.

**Comment by Rikaaaaaa95** [comment]
> Hello, I do not know what to do when the training progress finished, can you give me some advice? I'm a really new one for deep learning.

你好，方便留一下qq么

**Comment by louxiantuo** [comment]
> Thank you for your question.
> Indeed, make_dataset.py make only one dataset including both of train and test dataset.
> I divided it manually.

So，how to divided it?Which is train?Which is test?

**Comment by Ygya** [comment]
I want to ask you how to test this code and how can I get the final change detection results? Thank you!

**Comment by liudong315** [comment]
请问数据预处理后，应该如何划分训练集和测试集？
